### DISCUSSION TITLE
Possible to implement React using vanilla Vue? (then React Native using vanilla Vue?)

### DISCUSSION THREAD

**Discussion by cyfung1031** [vuejs/rfcs#583] [General]
As Vue is a framework, should it be able to imitate React APIs using Vue?
(vue's core, not .vue template)

Then we can have a high performance React powered by Vue. (no more `shouldComponentUpdate()`)
So we can move this into React Native to develop Mobile Apps?

PS. The webpack / bundle will make the dist output using vanilla Vue by translating JSX.

**Comment by sxzz** [comment]
See
- Run Vue component in React: https://github.com/antfu/reactivue
- Run React component in Vue: https://github.com/unixzii/revue
